scripts/update_specs_from_pricing.py

```diff
@@ -145,14 +115,13 @@ def get_mq_pricing():
         'mq.m5.2xl': 'mq.m5.2xlarge',
         'mq.m5.4xl': 'mq.m5.4xlarge'
```

Comment:
[`mq.m5.2xl` and `mq.m5.4xl` are the only ones present so far, but wondering if everything ending in `xl` should have `arge` appended like `Neptune`](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonMQ/current/index.json)